### PR TITLE
Fixes for /grant and claims

### PR DESCRIPTION
**Pull request by rubellyte**
- Allow /grant to be used by Admins
- Move automatic ship protection to claims
- .gitignore updated to ignore /config folder

### CODE REVIEW

File: plugins/player_manager.py [hunk 1]
```diff
@@ -548,6 +548,29 @@ def get_role(self, name):
...
+    def get_rank(self, player):
+        """
+        Returns the highest rank of the given player.
+
+        :param player: Player to check rank of.
+        :return: Int: A number representing the highest rank.
+        """
+        # Owner isn't properly listed in the player's roles, so check the
+        # owner uuid as well.
+        if player.check_role(Owner) or \
```
Comment (kharidiron) [inline plugins/player_manager.py:560]: Another way you can handle this if tree is to use an enum object, and then just do a enum lookup. Look in the utilities.py for an example of what I mean.

File: .gitignore [hunk 2]
```diff
@@ -36,7 +36,7 @@ nosetests.xml
 .pydevproject
 
 # Project specific
-/config/config.json
+/config/
```
Comment (kharidiron) [inline .gitignore]: We don't want to ignore the whole config folder... we actually want the config.json.defaults file to be copied over, along with the folder structure.

**Review by kharidiron — CHANGES_REQUESTED** [review]
All in all, I'm happy with the idea, and would definitely like to test it a bit first before pulling it in. However, the config folder change in gitignore needs to be changed first.

File: .gitignore (hunk 2 above)
Comment (rubellyte) [reply .gitignore]: Ah, yeah, that's true. Whoops.

File: plugins/player_manager.py (hunk 1 above)
Comment (kharidiron) [reply plugins/player_manager.py:560]: Yeah, in hindsight, disregard this. It isn't as workable as I originally thought...

**Review by kharidiron — APPROVED** [review]
Yup. This is better.

**Pull request merged**